### DISCUSSION TITLE
Button to add new finding to template

### DISCUSSION THREAD

**Discussion by cmahrl** [Syslifters/sysreptor#46] [General]
### Problem
Currently if you want to create a template from a new finding, one would have to manually create the template an copy each field of the finding.

### Solution
It should be possible to directly create a template from a new issue. E.g. a dialog should be opened that allows the user to remove sensitive customer specific data. On save, the template is created.

**Comment by richardschwabe** [comment]
I am currently writing my first real report with the software as well and this is the feature I am missing.

**Comment by Patralos** [comment]
This has been implemented with the latest release https://github.com/Syslifters/sysreptor/releases/tag/0.110
